src/AliceInventory/Controllers/ValuesController.cs

```diff
@@ -3,43 +3,21 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
 
 namespace AliceInventory.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class ValuesController : ControllerBase
     {
-        // GET api/values
-        [HttpGet]
-        public ActionResult<IEnumerable<string>> Get()
-        {
-            return new string[] { "value1", "value2", "value3" };
-        }
-
-        // GET api/values/5
-        [HttpGet("{id}")]
-        public ActionResult<string> Get(int id)
-        {
-            return "value";
-        }
 
         // POST api/values
         [HttpPost]
-        public void Post([FromBody] string value)
-        {
-        }
-
-        // PUT api/values/5
-        [HttpPut("{id}")]
-        public void Put(int id, [FromBody] string value)
-        {
-        }
-
-        // DELETE api/values/5
-        [HttpDelete("{id}")]
-        public void Delete(int id)
+        public ActionResult<string> Post([FromBody] AliceRequest request)
         {
+            var responce = new AliceResponse() {Response = new ResponseModel() {Text = request.Request.Command}};
```

Comment:
Typo in variable name "responce". Have to be "responSe". The same mistakes exist in other places, for example, in methods' signatures

Reply:
Already [fixed](https://github.com/vkamiansky/hackathonfls2019/commit/3bbd0963957d605cba5661a004e774b18f03db88)